telemetry/admin-metrics-export/src/main/scala/io/buoyant/telemetry/admin/AdminMetricsExportTelemeter.scala

```diff
@@ -122,3 +123,5 @@ class AdminMetricsExportTelemeter(
   }
 
 }
+
+object histogramSnapshotInterval extends GlobalFlag("60", "Interval, in seconds, to snapshot histrograms")
```

Comment:
You may just want to use a GlobalFlag[Duration] -- this allows specification of i.e. "60.seconds" "1.minute" etc -- and there's a default Flag handler for Duration so it all Just Works.